app/src/main/java/com/dashxdemo/app/application/MyApplication.kt

```diff
@@ -0,0 +1,11 @@
+package com.dashxdemo.app.application
+
+import android.app.Application
+import com.dashx.sdk.DashXClient
+
+class MyApplication: Application() {
+    override fun onCreate() {
+        super.onCreate()
+        DashXClient.createInstance(this,"TLy2w3kxf8ePXEyEjTepcPiq","https://api.dashx-staging.com/graphql","staging")
```

Comment:
Keys shouldn't be hard-coded. We should provide best practices -- how the user would configure this (per environment) in their application.